Build-V3-Plugin-Optimizers-ios/Assets/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyCollectedDataTypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyCollectedDataType</key>
```

Comment:
@v-pedro-bryj question related to privacy manifest. 

I saw that you specified the empty `NSPrivacyCollectedDataType`, I think we should have to provide the type of data we are collecting. If we are not, then we are safe to remove this dictionary. 

`NSPrivacyCollectedDataTypeLinked` & `NSPrivacyCollectedDataTypeTracking` are set to `false` but we are not specifying the NSPrivacyCollectedDataType. 

Apple's official document: [NSPrivacyCollectedDataType](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests#4144131)

<img width="1080" alt="Screenshot 2024-05-16 at 4 15 23 PM" src="https://github.com/bryjai/build-v3-plugin-optimizers-ios/assets/108368167/ecdd2046-d86a-4360-9971-bbd1b941570e">

Reply:
@piyush-bryj  as we don't have anything to add in the privacy manifest, we can follow this example 
https://github.com/SVProgressHUD/SVProgressHUD/blob/master/SVProgressHUD/PrivacyInfo.xcprivacy
what do you think?

Reply:
Yeah I agree. If we don't have anything to add then we can leave it empty. I don't think Apple will mind having those fields blank. 

Also our SDK's don't collect much or anything. It's up to the customer apps to declare what they are collecting using our SDKs

Reply:
@dhekra-zaied-bryj Yes, we can follow that example. Thanks for sharing it with us.

Reply:
Fixed!

Reply:
@v-pedro-bryj I think it's fine because I have seen empty files for many third-party SDKs.